docs/sources/datasources/opentsdb.md

```diff
@@ -23,6 +23,24 @@ To access OpenTSDB settings, hover your mouse over the **Configuration** (gear)
 | `Version`    | Version = opentsdb version, either <=2.1 or 2.2                                                                                       |
 | `Resolution` | Metrics from opentsdb may have datapoints with either second or millisecond resolution.                                               |
 | `Lookup Limit`| Default is 1000.                                                                                                                     |
+| `Min time interval`| Explained below.                                                                                                                     |
+
+- Min time interval
+
+A lower limit for the auto group by time interval. Recommended to be set to write frequency, for example `1m` if your data is written every minute.
+This option can also be overridden/configured in a dashboard panel under data source options. It's important to note that this value **needs** to be formatted as a
+number followed by a valid time identifier, e.g. `1m` (1 minute) or `30s` (30 seconds). The following time identifiers are supported:
+
```

Comment:
Add a descriptive heading (h3) such as Supported min time identifiers. Link to the heading from the table entry. 

Include the last sentence from the preceding paragraph as intro text below the new heading.

Reply:
Thank you for your time to review my code, I optimized the docs according to your suggestions.